Gems/RobotecSpectatorCamera/Code/Include/RobotecSpectatorCamera/RobotecSpectatorCameraBus.h

```diff
@@ -0,0 +1,33 @@
+
+#pragma once
+
+#include <RobotecSpectatorCamera/RobotecSpectatorCameraTypeIds.h>
+
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Interface/Interface.h>
+
+namespace RobotecSpectatorCamera
+{
+    class RobotecSpectatorCameraRequests
```

Comment:
It would be a good idea to put some useful interface here, like switching between modes.